CRITICAL - 2025-05-14 21:38:51 --> CodeIgniter\Database\Exceptions\DataException: Allowed fields must be specified for model: "App\Models\Subtarea"
[Method: POST, Route: guardar_subtarea]
in SYSTEMPATH\Model.php on line 822.
 1 SYSTEMPATH\Model.php(822): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields('App\\Models\\Subtarea')
 2 SYSTEMPATH\BaseModel.php(820): CodeIgniter\Model->doProtectFieldsForInsert([...])
 3 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 4 APPPATH\Controllers\Subtareas.php(26): CodeIgniter\Model->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-14 21:50:02 --> mysqli_sql_exception: Table 'activity_now.subtareas' doesn't exist in C:\xampp\htdocs\activity_now\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\activity_now\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `su...', 0)
#1 C:\xampp\htdocs\activity_now\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `su...')
#2 C:\xampp\htdocs\activity_now\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `su...')
#3 C:\xampp\htdocs\activity_now\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `su...', Array, false)
#4 C:\xampp\htdocs\activity_now\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\activity_now\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\activity_now\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\activity_now\app\Controllers\Subtareas.php(26): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
#9 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
#10 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\activity_now\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\activity_now\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\activity_now\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-05-14 21:50:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'activity_now.subtareas' doesn't exist
[Method: POST, Route: guardar_subtarea]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Subtareas.php(26): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-14 21:50:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'activity_now.subtareas' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Subtareas.php(26): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-14 21:50:02 --> [Caused by] mysqli_sql_exception: Table 'activity_now.subtareas' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Subtareas.php(26): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-14 21:50:29 --> mysqli_sql_exception: Column 't_id' cannot be null in C:\xampp\htdocs\activity_now\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\activity_now\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `su...', 0)
#1 C:\xampp\htdocs\activity_now\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `su...')
#2 C:\xampp\htdocs\activity_now\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `su...')
#3 C:\xampp\htdocs\activity_now\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `su...', Array, false)
#4 C:\xampp\htdocs\activity_now\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\activity_now\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\activity_now\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\activity_now\app\Controllers\Subtareas.php(25): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
#9 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
#10 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\activity_now\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\activity_now\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\activity_now\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-05-14 21:50:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 't_id' cannot be null
[Method: POST, Route: guardar_subtarea]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtarea` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Subtareas.php(25): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-14 21:50:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 't_id' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `subtarea` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `subtarea` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtarea` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Subtareas.php(25): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-14 21:50:29 --> [Caused by] mysqli_sql_exception: Column 't_id' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `subtarea` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `subtarea` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `subtarea` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (NULL, NULL, \'intento\', \'Pendiente\', \'2\', \'2025-05-14\', \'intneto1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtarea` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Subtareas.php(25): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-05-14 21:57:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-14 21:57:05 --> mysqli_sql_exception: Table 'activity_now.subtareas' doesn't exist in C:\xampp\htdocs\activity_now\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\activity_now\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `su...', 0)
#1 C:\xampp\htdocs\activity_now\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `su...')
#2 C:\xampp\htdocs\activity_now\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `su...')
#3 C:\xampp\htdocs\activity_now\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `su...', Array, false)
#4 C:\xampp\htdocs\activity_now\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\activity_now\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\activity_now\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\activity_now\app\Controllers\Subtareas.php(24): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
#9 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
#10 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\activity_now\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\activity_now\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\activity_now\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-05-14 21:57:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'activity_now.subtareas' doesn't exist
[Method: POST, Route: guardar_subtarea]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Subtareas.php(24): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-14 21:57:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'activity_now.subtareas' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (\'1\', NULL, \'inento1\', \'En Progreso\', \'2\', \'2025-05-14\', \'intento2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (\'1\', NULL, \'inento1\', \'En Progreso\', \'2\', \'2025-05-14\', \'intento2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Subtareas.php(24): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-14 21:57:05 --> [Caused by] mysqli_sql_exception: Table 'activity_now.subtareas' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (\'1\', NULL, \'inento1\', \'En Progreso\', \'2\', \'2025-05-14\', \'intento2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (\'1\', NULL, \'inento1\', \'En Progreso\', \'2\', \'2025-05-14\', \'intento2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (\'1\', NULL, \'inento1\', \'En Progreso\', \'2\', \'2025-05-14\', \'intento2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `subtareas` (`t_id`, `u_id`, `st_descripcion`, `st_estado`, `st_prioridad`, `st_fechaVenc`, `st_comentario`) VALUES (:t_id:, :u_id:, :st_descripcion:, :st_estado:, :st_prioridad:, :st_fechaVenc:, :st_comentario:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Subtareas.php(24): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Subtareas->guardar_subtarea()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Subtareas))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-05-14 21:57:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:18:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:19:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:19:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:19:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:20:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:20:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:20:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:23:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:23:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:24:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:29:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:29:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:29:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:29:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:34:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:34:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:34:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:36:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:37:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:37:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 22:37:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:11:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-05-14 23:11:11 --> TypeError: form_textarea(): Argument #2 ($value) must be of type string, null given, called in C:\xampp\htdocs\activity_now\app\Views\tareas\ver_tarea.php on line 92
[Method: GET, Route: ver_tarea/1]
in SYSTEMPATH\Helpers\form_helper.php on line 211.
 1 APPPATH\Views\tareas\ver_tarea.php(92): form_textarea([...], null)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\activity_now\\app\\Views\\tareas\\ver_tarea.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('tareas/ver_tarea', [], true)
 5 APPPATH\Controllers\Tareas.php(169): view('tareas/ver_tarea', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-05-14 23:11:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-05-14 23:11:20 --> TypeError: form_textarea(): Argument #2 ($value) must be of type string, null given, called in C:\xampp\htdocs\activity_now\app\Views\tareas\ver_tarea.php on line 92
[Method: GET, Route: ver_tarea/1]
in SYSTEMPATH\Helpers\form_helper.php on line 211.
 1 APPPATH\Views\tareas\ver_tarea.php(92): form_textarea([...], null)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\activity_now\\app\\Views\\tareas\\ver_tarea.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('tareas/ver_tarea', [], true)
 5 APPPATH\Controllers\Tareas.php(169): view('tareas/ver_tarea', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-05-14 23:11:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-05-14 23:11:24 --> TypeError: form_textarea(): Argument #2 ($value) must be of type string, null given, called in C:\xampp\htdocs\activity_now\app\Views\tareas\ver_tarea.php on line 92
[Method: GET, Route: ver_tarea/1]
in SYSTEMPATH\Helpers\form_helper.php on line 211.
 1 APPPATH\Views\tareas\ver_tarea.php(92): form_textarea([...], null)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\activity_now\\app\\Views\\tareas\\ver_tarea.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('tareas/ver_tarea', [], true)
 5 APPPATH\Controllers\Tareas.php(169): view('tareas/ver_tarea', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-05-14 23:12:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-05-14 23:12:02 --> TypeError: form_textarea(): Argument #2 ($value) must be of type string, null given, called in C:\xampp\htdocs\activity_now\app\Views\tareas\ver_tarea.php on line 92
[Method: GET, Route: ver_tarea/1]
in SYSTEMPATH\Helpers\form_helper.php on line 211.
 1 APPPATH\Views\tareas\ver_tarea.php(92): form_textarea([...], null)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\activity_now\\app\\Views\\tareas\\ver_tarea.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('tareas/ver_tarea', [], true)
 5 APPPATH\Controllers\Tareas.php(169): view('tareas/ver_tarea', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-05-14 23:12:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-05-14 23:12:09 --> TypeError: form_textarea(): Argument #2 ($value) must be of type string, null given, called in C:\xampp\htdocs\activity_now\app\Views\tareas\ver_tarea.php on line 92
[Method: GET, Route: ver_tarea/1]
in SYSTEMPATH\Helpers\form_helper.php on line 211.
 1 APPPATH\Views\tareas\ver_tarea.php(92): form_textarea([...], null)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\activity_now\\app\\Views\\tareas\\ver_tarea.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('tareas/ver_tarea', [], true)
 5 APPPATH\Controllers\Tareas.php(169): view('tareas/ver_tarea', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-05-14 23:17:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:17:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:17:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:17:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:18:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:18:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:18:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:18:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:19:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-05-14 23:24:55 --> mysqli_sql_exception: Unknown column 'st_tarea_id' in 'where clause' in C:\xampp\htdocs\activity_now\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\activity_now\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\activity_now\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\activity_now\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\activity_now\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\activity_now\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\activity_now\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\activity_now\app\Controllers\Tareas.php(164): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
#8 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
#9 C:\xampp\htdocs\activity_now\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\activity_now\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\activity_now\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\activity_now\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-14 23:24:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'st_tarea_id' in 'where clause'
[Method: GET, Route: ver_tarea/1]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `subtarea`
WHERE `st_tarea_id` = :st_tarea_id:', [...], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Tareas.php(164): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-14 23:24:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'st_tarea_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `subtarea`
WHERE `st_tarea_id` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `subtarea`
WHERE `st_tarea_id` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `subtarea`
WHERE `st_tarea_id` = :st_tarea_id:', [...], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Tareas.php(164): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-14 23:24:55 --> [Caused by] mysqli_sql_exception: Unknown column 'st_tarea_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `subtarea`
WHERE `st_tarea_id` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `subtarea`
WHERE `st_tarea_id` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `subtarea`
WHERE `st_tarea_id` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `subtarea`
WHERE `st_tarea_id` = :st_tarea_id:', [...], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Tareas.php(164): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Tareas->ver_tarea('1')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Tareas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-05-14 23:26:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:27:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:29:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:29:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:49:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:49:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:50:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:50:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:51:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:51:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-05-14 23:51:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
